src/Features/PostProcessing/BokehResources.cpp

```
@@ -48,31 +59,177 @@ bool BokehResources::LoadTextureFromFile(const std::filesystem::path& path, int
+				   image.GetImages(),
+				   image.GetImageCount(),
+				   image.GetMetadata(),
+				   DXGI_FORMAT_R32G32B32A32_FLOAT,
+				   DirectX::TEX_FILTER_DEFAULT,
+				   0.0f,
+				   converted))) {
+		return false;
+	}
+
+	const DirectX::Image* source = converted.GetImage(0, 0, 0);
+	if (!source || source->width == 0 || source->height == 0)
```

Cppcheck (2.21.0)

[error] Returning pointer that will be invalid when returning.

(returnDanglingLifetime)